pages/tissue-summary/[taxa].tsx

```diff
@@ -0,0 +1,490 @@
+  "homo-sapiens": {
+    dataProviderPath:
+      "/tissue-homo-sapiens/?type=Tissue&taxa=Homo+sapiens&virtual=false",
+    pagePath: "/tissue-summary/homo-sapiens",
+    queryAtType: "TissueSummaryHomoSapiens",
+    columnHeaderCellClass: "bg-tissue-matrix-column-header-homo-sapiens",
+    rowHeaderCellClass: "bg-tissue-matrix-row-header-homo-sapiens",
+    rowSubheaderCellClass: "bg-tissue-matrix-row-subheader-homo-sapiens",
+    dataCellClass: "bg-tissue-matrix-data-homo-sapiens",
+    hoverCellClass: "group-hover:bg-tissue-matrix-highlight-homo-sapiens",
+  },
+  "mus-musculus": {
+    dataProviderPath:
+      "/tissue-mus-musculus/?type=Tissue&taxa=Mus+musculus&virtual=false",
+    pagePath: "/tissue-summary/mus-musculus",
+    queryAtType: "TissueSummaryMusMusculus",
+    columnHeaderCellClass: "bg-tissue-matrix-column-header-mus-musculus",
+    rowHeaderCellClass: "bg-tissue-matrix-row-header-mus-musculus",
+    rowSubheaderCellClass: "bg-tissue-matrix-row-subheader-mus-musculus",
+    dataCellClass: "bg-tissue-matrix-data-mus-musculus",
+    hoverCellClass: "group-hover:bg-tissue-matrix-highlight-mus-musculus",
+  },
+};
+
+/**
+ * List all accepted taxa.
+ */
+const acceptedTaxa = Object.keys(taxaQueries);
+
+/**
+ * Generate a map of column labels to their 0-based column index. Each label appears as a key in
+ * the returned object, with the value being the index of the corresponding column in the matrix.
+ * @param columnBuckets - Buckets for the x-axis of the matrix
+ * @returns Map of column labels to their 0-based column index
+ */
+function generateMatrixColumnMap(columnBuckets: MatrixBucket[]): ColumnMap {
+  const sortedBuckets = _.sortBy(columnBuckets, (bucket) =>
+    bucket.key.toLowerCase()
+  );
+  return sortedBuckets.reduce((acc, bucket, i) => {
+    const column = { [bucket.key]: i };
+    return { ...acc, ...column };
+  }, {});
+}
+
+/**
+ * Renders the top-level header cells that covers potentially multiple rows in the matrix. It
+ * includes a link to the corresponding list-view page.
+ * @param source - Source of data to render the cell
+ * @param meta - Metadata for the entire matrix to render the cell
+ */
+function RowHeaderCell({
+  source,
+  meta,
+}: {
+  source: CellSource;
+  meta: { taxa: string };
+}) {
+  const bgClass = taxaQueries[meta.taxa].rowHeaderCellClass;
+
+  return (
+    <Link
+      href={source.href}
+      className={`flex h-full items-center px-2 py-0.5 font-semibold no-underline ${bgClass}`}
+    >
+      {source.key}
+    </Link>
+  );
+}
+
+/**
+ * Renders the second-level header cells that covers a single row in the matrix.
+ * @param source - Source of data to render the cell
+ * @param meta - Metadata for the entire matrix to render the cell
+ */
+function RowSubheaderCell({
+  source,
+  meta,
+}: {
+  source: CellSource;
+  meta: { taxa: string };
+}) {
+  const bgClass = taxaQueries[meta.taxa].rowSubheaderCellClass;
+  const hoverClass = taxaQueries[meta.taxa].hoverCellClass;
```

Comment:
Is it possible to add hoverClass to RowHeaderCell and ColumnHeaderCell? Nobody wants it?

Reply:
People said that the row header cell hover was optional. `RowHeaderCell` can span multiple rows, so that would have been difficult to do without either making a custom version of `DataGrid` specifically for this page, or adding this generically in `DataGrid` which will take some effort I think.

I’ll need to do some generic modifications to `DataGrid` for future matrices (like sticky headers) so this ability might happen then, or possibly because it wasn’t seen as important, whereas the sub-row hover was seen as very important.